zipkin-storage/elasticsearch-http/src/test/java/zipkin/storage/elasticsearch/ElasticsearchDependenciesTest.java

```diff
@@ -14,6 +14,8 @@
 package zipkin.storage.elasticsearch;
 
 import com.google.common.base.Throwables;
+import java.io.IOException;
+import java.util.List;
```

Comment:
right now we use square code style (noted in the CONTRIBUTING section), and there's a PR for eclipse as yet unmerged https://github.com/square/java-code-styles/pull/32